MarsUpdated/Utilities/CommonDriver.cs

```diff
@@ -10,7 +10,7 @@ namespace MarsUpdated;
 [Binding]
 public class CommonDriver
 {
-    public static IWebDriver? driver;
+    public static IWebDriver driver;
 
 
```

Comment:
and do something like this here

public class CommonDriver
    {
        public static IWebDriver driver;


        public static void Initialize()
        {
            driver = new ChromeDriver();
            driver.Manage().Window.Maximize();
            LoginPage loginPageObj = new LoginPage();
            loginPageObj.LoginSteps(driver);


        }

        public static void Waits()
        {
            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(60);

        }

Reply:
Thanks Aman .. I will do..